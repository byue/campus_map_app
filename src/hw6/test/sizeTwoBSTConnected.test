# Simple test of breadth-first search algorithm.
# Verifies output of BFS for size 2 connected graph.

CreateGraph characters
AddNode characters GORILLA-MAN
AddNode characters HAWK

AddEdge characters GORILLA-MAN HAWK COMIC1

FindPath characters GORILLA-MAN HAWK

FindPath characters HAWK GORILLA-MAN 

FindPath characters GORILLA-MAN GORILLA-MAN

FindPath characters GORILLA-MAN G32

FindPath characters G31 G32