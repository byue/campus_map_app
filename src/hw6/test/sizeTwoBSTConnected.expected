# Simple test of breadth-first search algorithm.
# Verifies output of BFS for size 2 connected graph.

created graph characters
added node GORILLA-MAN to characters
added node HAWK to characters

added edge COMIC1 from GORILLA-MAN to HAWK in characters

path from GORILLA-MAN to HAWK:
GORILLA-MAN to HAWK via COMIC1

path from HAWK to GORILLA-MAN:
no path found

path from GORILLA-MAN to GORILLA-MAN:

unknown character G32

unknown character G31
unknown character G32